tests/mocks/pose.py

```diff
@@ -0,0 +1,14 @@
+from robot_interface.models.geometry.frame import Frame
+from robot_interface.models.geometry.orientation import Orientation
+from robot_interface.models.geometry.pose import Pose
+from robot_interface.models.geometry.position import Position
+
+
+class MockPose:
+    default_pose = Pose(
+        position=Position(x=1, y=1, z=1, frame=Frame.Robot),
+        orientation=Orientation(  # Rotated 90 degrees about z-axis
+            x=0, y=0, z=0.7071068, w=0.7071068, frame=Frame.Robot
+        ),
```

Comment:
```suggestion
        position=Position(x=0, y=0, z=0, frame=Frame.Robot),
        orientation=Orientation( 
            x=0, y=0, z=0, w=1, frame=Frame.Robot
        ),
```

Maybe this is slightly easier?

Reply:
I agree. Currently, no reason not to use the simplest mock for pose.